content/release-10-18-0/digital-twin-manager-10-18-0-bundle/10-18-0-1.md

```diff
@@ -0,0 +1,355 @@
+<td>A generic tree view of entities has been created for the <b>Asset models</b> page.</td>
+<td>CTM-446</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Lazy loading has been implemented for the tree view in the <b>Assets</b> page and <b>Asset tree</b> tab. The "Expand all" and "Collapse all" functionality has been removed.</td>
+<td>CTM-553</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now import the selected asset models into a tenant.</td>
+<td>CTM-430</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The <b>Export asset model</b> popup now shows a tree view representation of all the asset models.</td>
+<td>CTM-444</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr
+
+<tr>
+<td>Users can see a preview of selected asset models that will be imported.</td>
+<td>CTM-484</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr
+
+<tr>
+<td>Users can now import the selected asset properties into a tenant.</td>
+<td>CTM-431</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can see a preview of selected asset properties that will be imported.</td>
+<td>CTM-581</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now export the selected asset models from a tenant.</td>
+<td>CTM-428</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now export the selected asset properties from a tenant.</td>
+<td>CTM-429</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now import or export a maximum of 250 asset models and 500 asset properties.</td>
+<td>CTM-454</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Users can now move the child assets to another asset which belongs to the same asset model.</td>
+<td>CTM-491</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>A breadcrumb is shown for the <b>Asset model samples</b> page.</td>
+<td>CTM-563</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset tree</b> tab, the count for the number of asset models has been removed.</td>
+<td>CTM-559</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset tree</b> tab, a <b>Reload</b> button is now displayed.</td>
+<td>CTM-555</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The import assets layout has been modified to be in sync with other layouts.</td>
+<td>CTM-550</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Localizations have been added for the <b>Subassets</b> page.</td>
+<td>CTM-373</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Cell renderers have been added for child asset models and asset properties columns in the <b>Export Asset models</b> popup.</td>
+<td>CTM-583</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>In the <b>Asset model</b> creation page, the <b>New asset property</b> button is always enabled and is by default assigned to the asset model after creation.</td>
```

Comment:
@spau6 Does this entry refer to the **Asset properties** page and the **Add asset property** button?

Reply:
It is Asset Model creation page wherein we will associate a Asset property (which is already created) to that Asset model.
If the property is not created earlier then user can create it using Add asset property button available under property selection section

Reply:
![image](https://github.com/SoftwareAG/c8y-releasenotes/assets/109800638/e38f9350-28ae-4d7e-b946-baf081c4c32f)